game/tests.py

```
@@ -1631,3 +1631,59 @@ def fake_engine(cmd):
+    def test_delete_account_get_requires_login(self):
+        response = self.client.get('/delete-account/')
+        self.assertEqual(response.status_code, 302)
+
+    def test_delete_account_get_logged_in(self):
+        self.client.login(username='deleteme', password='Password123!')
+        response = self.client.get('/delete-account/')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'game/delete_account.html')
+
+    def test_delete_account_post_invalid_credentials(self):
+        self.client.login(username='deleteme', password='Password123!')
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)